pages/common/vault.md

```diff
@@ -0,0 +1,27 @@
+# vault
+
+> A CLI to interact with HashiCorp Vault.
+
+- Create a new vault, requiring at least 2 out of 5 keyshares to unseal:
+
+`vault init -key-shares={{5}} -key-threshold={{2}}`
+
+- Unseal the vault by providing one of the keyshares. Repeat with necessary key-shares until unsealed:
+
+`vault unseal {{key-share-x}}`
+
+- Authenticate client against vault, using an authentication token:
+
+`vault auth {{authentication-token}}`
+
+- Store a new secret in the vault using the generic back-end called secret:
+
+`vault write secret/{{hello}} value={{world}}`
+
+- Read a value from the vault using the generic back-end called secret:
+
+`vault read secret/{{hello}}`
+
+- Seal the vault again:
```

Comment:
Here we could take the opportunity that the description is short, to explain what seal/unseal does exactly. E.g., does this command unmount a folder, encrypt a file, mark stuff as read-only, or something else?

Reply:
It deletes the master encryption key from memory, rendering all data inaccessible until someone *unseals* (or *unlocks*) the vault again.